pkg/cmd/kafka/create/api_validators.go

```diff
@@ -119,3 +121,19 @@ func (input *ValidatorInput) ValidateSize() error {
 
 	return nil
 }
+
+// ValidateBillingModel validates if user provided a correct billing model
+func ValidateBillingModel(billingModel string) error {
```

Comment:
Oh.. wow. I would not go that deep to validate those. If you do not specify valid billing model you would not get quota. It is great UX so thank you for this